cypress/support/commands.ts

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable import/no-unresolved */
+import '@testing-library/cypress/add-commands'
+/* eslint-enable import/no-unresolved */
```

Comment:
Added this to fix the [Cypress function not found issue](https://github.com/nimblehq/react-templates/issues/31) on React template